Rakefile

```
@@ -27,143 +27,140 @@ namespace :test do
+      # beaker_gem_dir = beaker_gem_spec.gem_dir
+      # beaker_test_base_dir = File.join(beaker_gem_dir, 'acceptance/tests/base')
+      # load_path_option = File.join(beaker_gem_dir, 'acceptance/lib')
 
       ENV['BEAKER_setfile'] = 'acceptance/config/nodes/hosts.yaml'
-      sh("beaker",
-         "--hosts", "acceptance/config/nodes/hosts.yaml",
-          # We can't run these tests until the rsync support in the main
-          # beaker/host.rb is updated to work with passwords.
-          # "--tests", beaker_test_base_dir,
-          # "--load-path", load_path_option,
-         "--tests", 'acceptance/tests/',
-         "--log-level", "debug",
-         "--debug")
+      sh('beaker',
+         '--hosts', 'acceptance/config/nodes/hosts.yaml',
+         ## We can't run these tests until the rsync support in the main
+         ## beaker/host.rb is updated to work with passwords.
+         # '--tests', beaker_test_base_dir,
+         # '--load-path', load_path_option,
+         '--tests', 'acceptance/tests/',
+         '--log-level', 'debug',
+         '--debug')
     end
   end
 end
 
 # namespace-named default tasks.
 # these are the default tasks invoked when only the namespace is referenced.
 # they're needed because `task :default` in those blocks doesn't work as expected.
-task 'test:spec' => 'test:spec:run'
-task 'test:acceptance' => 'test:acceptance:quick'
+task 'test:spec': %i[test:spec:run]
+task 'test:acceptance': %i[test:acceptance:quick]
 
 # global defaults
-task :lint => %i[lint:rubocop]
-task :test => 'test:spec'
-task :default => :test
+task lint: %i[lint:rubocop]
+task test: %i[test:spec]
+task default: %i[test]
 
 ###########################################################
 #
 #   Documentation Tasks
 #
 ###########################################################
-DOCS_DAEMON = "yard server --reload --daemon --server thin"
+DOCS_DAEMON = 'yard server --reload --daemon --server thin'
 FOREGROUND_SERVER = 'bundle exec yard server --reload --verbose --server thin lib/beaker'
 
-def running?( cmdline )
+def running?(cmdline)
   ps = `ps -ef`
-  found = ps.lines.grep( /#{Regexp.quote( cmdline )}/ )
-  if found.length > 1
-    raise StandardError, "Found multiple YARD Servers. Don't know what to do."
-  end
+  found = ps.lines.grep(/#{Regexp.quote(cmdline)}/)
+  raise StandardError, "Found multiple YARD Servers. Don't know what to do." if found.length > 1
 
-  yes = found.empty? ? false : true
-  return yes, found.first
+  [!found.empty?, found.first]
 end
 
-def pid_from( output )
-  output.squeeze(' ').strip.split(' ')[1]
+def pid_from(output)
+  output.squeeze(' ').strip.split[1]
 end
 
 desc 'Start the documentation server in the foreground'
-task :docs => 'docs:clear' do
+task docs: %i[docs:clear] do
   original_dir = Dir.pwd
-  Dir.chdir( File.expand_path(File.dirname(__FILE__)) )
+  Dir.chdir(__dir__)
   sh FOREGROUND_SERVER
-  Dir.chdir( original_dir )
+  Dir.chdir(original_dir)
```

Perhaps even better to use it as a block?
```suggestion
  Dir.chdir(__dir__) do
    sh FOREGROUND_SERVER
  end
```

---

Ah, nice! Didn't know it works this way! Let me implement it